requirements-py36.txt

```diff
@@ -0,0 +1,27 @@
+pandas==1.1.5
```

Comment:
This file should probably be located near to `requirements/environment-py36.yml`.

Reply:
this is questionable, we don't have `pip` files in there, so I just placed new `.txt` alongside existing one